_docs_integrate/migration-from-v4-to-v5.md

```diff
@@ -16,7 +16,7 @@ properties:
   - level:
   - implementation status: DOCS ONLY
   - documentation status: DONE
-  - published: true
+  - published: true 
```

Comment:
urgh, does the cli generate trailing whitespaces??

Reply:
There was a trailing whitespace in the excel.